tests/core_tests.py

```diff
@@ -354,13 +354,57 @@ def test_queries_endpoint(self):
         self.assertEquals(403, resp.status_code)
 
     def test_search_query_endpoint(self):
-        userId = 'userId=null'
-        databaseId = 'databaseId=null'
-        searchText = 'searchText=null'
-        status = 'status=success'
-        params = [userId, databaseId, searchText, status]
-        resp = self.client.get('/caravel/search_queries?'+'&'.join(params))
-        self.assertEquals(200, resp.status_code)
+        self.login('admin')
+        self.run_sql("SELECT * FROM ab_user", 'admin', client_id='client_id_1')
```

Comment:
move query creation into the setUp function:
https://docs.python.org/2/library/unittest.html